site/docs/v1/tech/apis/users.adoc

```diff
@@ -492,7 +492,7 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 Available Since Version 1.19.0
 ====
 
-THis API is used to import refresh tokens from an external system, this would generally be done during an initial user import or as an auxiliary step during a migration strategy.
+This API is used to import refresh tokens from an external system, this would generally be done during an initial user import or as an auxiliary step during a migration strategy.
```

Comment:
/docs/v1/tech/integrations/twilio/ needs to be removed or marked deprecated, too.

Reply:
It looks like this is being handled in this PR: https://github.com/FusionAuth/fusionauth-site/pull/590 so I'm going to hold off.